cmd/ssm-run/README.md

```diff
@@ -40,42 +40,46 @@ If a region is not specified, you will receive the following warning, but execut
 WARNING	No AWS region setting found. Will default to the region linked to any profile in use.
 ```
 
-The use of the `--limit` flag, if set, will result in your command being executed on the specified number of random instances returned for each profile/region combination. For example, if you set `--limit=10` on the account `foo` for regions `us-east-1` and `us-west-2`, your command would be executed on up to 20 total instances.
-
 #### running command(s) on a single instance
 
 ```
-> ./ssm-run -i i-12345 -c "uname -a"
-
-INFO    Command(s) to be executed: uname -a
-INFO    Fetched 1 instances for account [profile1] in [us-east-1].
-INFO    Instance ID              Region          Profile         Status
-INFO    i-12345					 us-east-1       profile1         Success
+> ./ssm run -p 'profile1' -i i-12345 -c 'uname'
+INFO    Command(s) to be executed:
+uname             
+INFO    Started invocation f73f2225-8fb2-4e63-ba63-6e2af54b8659 for profile1 in us-east-1 
+INFO    Instance ID              Region          Profile         Status 
+INFO    i-12345                  us-east-1       profile1        Success 
+INFO    Linux                                        
 INFO    Execution results: 1 SUCCESS, 0 FAILED
 ```
 
 Running multiple commands is easy; just separate your commands with semicolons (;). These commands will be executed in series, in the same console context (e.g., variables you set can be used between commands).
 
 ```
-> ./ssm-run -i i-12345 -c "uname -a; hostname; ifconfig"
-
-INFO    Command(s) to be executed: uname -a, hostname, ifconfig
-INFO    Fetched 1 instances for account [profile1] in [us-east-1].
-INFO    Instance ID              Region          Profile         Status
-INFO    i-12345					 us-east-1       profile1         Success
-INFO    Execution results: 1 SUCCESS, 0 FAILED
+> ./ssm run -p 'profile1' -i i-12345 -c 'uname;uname;uname'
+INFO    Command(s) to be executed:
+uname
+uname
+uname 
+INFO    Started invocation 73ee2f4c-fd54-4505-8ef7-1bb2baecd64f for profile1 in us-east-1 
+INFO    Instance ID              Region          Profile         Status 
+INFO    i-12345                  us-east-1       profile1        Success 
+INFO    Linux
+Linux
+Linux                            
+INFO    Execution results: 1 SUCCESS, 0 FAILED 
 ```
 
 #### running command(s) on multiple instances
 
 ```
-> ./ssm-run -i i-12345,i-67890 -c "uname -a"
-
-INFO    Command(s) to be executed: uname -a
-INFO    Fetched 2 instances for account [profile1] in [us-east-1].
-INFO    Instance ID              Region          Profile         Status
-INFO    i-12345					 us-east-1       profile1         Success
-INFO    i-67890					 us-east-1       profile1         Success
+> ./ssm run -p 'profile1' -i 'i-0879fe217fe11ad86,i-062aede5be23eef7a' -c 'uname > /dev/null 2>&1'
```

Comment:
Should we show the output of the command instead of just piping to `/dev/null` so as to illustrate what that would look like interspersed with the status lines for each instance?